app_customers/lib/util/drawer_controller.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:get/get.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+/// Drawer menu controller.
+class CustomDrawerController extends GetxController {
+  Rx<int> value = Rx<int>(0);
```

Comment:
What does value represent??

Reply:
its in this variable that i put the activation value of drawer page

Reply:
Add a comment and clearly define the variable name. `value` as the name of the variable may be take confusion and make the code incomprehensible.Please read the https://dart.dev/effective-dart/documentation